README.md

```diff
@@ -32,17 +32,17 @@ route won't get called, resulting in a 404 error.
 
 Rocket is extensively documented:
 
-  * [Overview]: A brief look at what makes Rocket special.
-  * [Quickstart]: How to get started as quickly as possible.
-  * [Getting Started]: How to start your first Rocket project.
-  * [Guide]: A detailed guide and reference to Rocket.
-  * [API Documentation]: The "rustdocs".
-
-[Quickstart]: https://rocket.rs/guide/quickstart
-[Getting Started]: https://rocket.rs/guide/getting-started
-[Overview]: https://rocket.rs/overview/
-[Guide]: https://rocket.rs/guide/
-[API Documentation]: https://api.rocket.rs/rocket/
+- [Overview]: A brief look at what makes Rocket special.
+- [Quickstart]: How to get started as quickly as possible.
+- [Getting Started]: How to start your first Rocket project.
+- [Guide]: A detailed guide and reference to Rocket.
+- [API Documentation]: The "rustdocs".
+
+[quickstart]: https://rocket.rs/guide/quickstart
```

Comment:
Do the links still work even if the case is mismatched? This looks like either a net neutral or net negative change, depending on the parser.

Reply:
Yes, they still working fine. It can be checked at the forked repo: https://github.com/silvioprog/Rocket